lab-jaren/lib/read-many-files.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const fs = require('fs');
+const async = require('async');
+
+module.exports = function(filePaths, callback) {
+  var result = [];
+  async.eachOf(filePaths, function(path, index){
```

Comment:
Good job figuring out how to use this new module's syntax. I can totally see you studied the shit out of those docs. 

Again, as to controlling the async, I'm not too sure myself, but it's a good thing to bring up with Duncan when you have time. 
 
I think you did a great job figuring this out either way, in spite of the async issue.